app/models/reports/schema_v2.rb

```diff
@@ -176,15 +118,33 @@ def registered_patients_query_v2(region)
       end
     end
 
-    private
+    memoize def missed_visits(with_ltfu: false)
+      field = with_ltfu ? :missed_visit_lost_to_follow_up : :missed_visit_under_care
```

Comment:
Nice, is this worth extracting? It would be the same for all treatment control fields no? It could even go into `FacilityState`: `facility_state.field(field_name, care_state)`?